client/src/components/atoms/InlineComment.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <q-card flat square class="bg-grey-1">
+    <q-card-section>
+      <div class="row items-start justify-between content-stretch no-wrap">
+        <div class="col-grow">
+          <div class="row items-center">
+            <div style="width: 30px">
+              <avatar-image :user="user" round class="fit" />
+            </div>
+            <div class="text-subtitle1 q-pl-sm">Magna Fringilla</div>
+          </div>
+          <div>
+            <small>February 17th, 2021 at 6:35pm</small>
+          </div>
+        </div>
+
+        <div class="col-auto">
+          <q-btn color="grey-7" round flat icon="more_vert">
+            <q-menu cover auto-close>
+              <q-list>
+                <q-item clickable>
+                  <q-item-section>Remove Card</q-item-section>
+                </q-item>
+                <q-item clickable>
+                  <q-item-section>Send Feedback</q-item-section>
+                </q-item>
+                <q-item clickable>
+                  <q-item-section>Share</q-item-section>
+                </q-item>
+              </q-list>
+            </q-menu>
+          </q-btn>
+        </div>
+      </div>
+    </q-card-section>
+
+    <q-card-section class="q-py-none">
+      <p>
+        At quis risus sed vulputate odio. Aliquam eleifend mi in nulla. Ornare
+        arcu odio ut sem nulla pharetra diam sit amet. Nulla pharetra diam sit
+        amet. Faucibus ornare suspendisse sed nisi lacus sed. Commodo quis
+        imperdiet massa tincidunt nunc pulvinar sapien. Egestas tellus rutrum
+        tellus pellentesque eu tincidunt tortor.
+      </p>
+    </q-card-section>
+
+    <div class="q-px-sm">
+      <q-chip size="16px" icon="bookmark"> Relevance </q-chip>
+      <q-chip size="16px" icon="bookmark"> Accessibility </q-chip>
+      <q-chip size="16px" icon="bookmark"> Coherence </q-chip>
+    </div>
+
+    <q-card-actions class="q-pa-md">
+      <q-btn bordered color="primary">Reply</q-btn>
+    </q-card-actions>
+
+    <q-separator />
+  </q-card>
+</template>
+
+<script setup>
+import AvatarImage from "./AvatarImage.vue"
+defineProps({
+  user: {
+    type: Object,
+    default: () => {},
+  },
+})
```

Comment:
```suggestion
const user = {}
```
Since this won't be coming in as a prop (rather "comment" will) I'd make this a var rather than a prop just to avoid confusing ourself when we implement into thinking this needs to be a prop.